main/src/main/js/start/src/components/Diff.js

```diff
@@ -0,0 +1,94 @@
+// Diff.js
+import React, { forwardRef } from "react";
+
+import { Button } from "react-materialize";
+
+import Icon from "react-materialize/lib/Icon";
+import Modal from "react-materialize/lib/Modal";
+
+import { Grid } from "@material-ui/core";
+
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { darcula } from "react-syntax-highlighter/dist/esm/styles/prism";
+import { prism } from "react-syntax-highlighter/dist/esm/styles/prism";
+
+import { capitalize } from "../utility";
+
+const Diff = (
+  { diff, lang, build, theme = "light", disabled, onLoad, onClose },
+  ref
+) => {
+  const onModalClose = () => {
```

Comment:
If you're not doing anything here you can just pass in `onClose` directly to the `options.onCloseStart`.